### PR TITLE
Update pt_br.json

### PR DESCRIPTION
**Pull request by VinnyStalck**

### CODE REVIEW

**Comment by mezz** [comment]
Sorry I missed this.
@DaDodger does this look good to you?

File: src/main/resources/assets/jei/lang/pt_br.json [hunk 1]
```diff
@@ -42,8 +42,8 @@
   "key.jei.showRecipe": "Mostrar receita",
   "key.jei.showUses": "Mostrar usos",
   "key.jei.recipeBack": "Mostrar receita anterior",
-  "key.jei.toggleCheatMode": "Ativar/desativar modo trapaça",
-   "key.jei.toggleEditMode": "Ocultar/editar modo",
+  "key.jei.toggleCheatMode": "Alternar modo trapaça",
+  "key.jei.toggleEditMode": "Alternar modo edição/ocultar",
```
Comment (ghost) [inline src/main/resources/assets/jei/lang/pt_br.json:46]: good fix

**Review by ghost — APPROVED** [review]
@mezz yeah, pretty fine

**Comment by mezz** [comment]
Thanks!

**Pull request merged**